index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <section class="hero">
+    <div class="container1">
+        <div class="top">
+          <h1>Header Logo</h1>
+          <ul class="top-links">
+            <li><a href="#">header link one</a></li>
+            <li><a href="#">header link two</a></li>
+            <li><a href="#">header link three</a></li>
+          </ul>
+        </div>
+        <div class="bottom">
+            <div class="text">
+              <h1>This website is awesome</h1>
+              <p>This website has some subtext that goes here under the main title. It's a smaller font and the color is lower contrast</p>
+              <a href="https://www.youtube.com/watch?v=uHgt8giw1LY">
+                <button class="button">Sign up</button>
+              </a>
+            </div>
+            <div class="hero-image">this is a placeholder for an image</div>
+        </div>
+    </div>
+  </section>
+
+    <section class="first">
+        <div class="container2">
+            <h1 class="section-title">Some random information.</h1>
+          <div class="images">  
+            <figure>
+                <div class="blank-img"></div>
+                <figcaption>this is some subtext under an illustration or image</figcaption>
+            </figure>
+            <figure>
+                <div class="blank-img"></div>
+                <figcaption>this is some subtext under an illustration or image</figcaption>
+            </figure>
+            <figure>
+                <div class="blank-img"></div>
+                <figcaption>this is some subtext under an illustration or image</figcaption>
+            </figure>
+            <figure>
+                <div class="blank-img"></div>
+                <figcaption>this is some subtext under an illustration or image</figcaption>
+            </figure>
+          </div>
+        </div>
+    </section>
+
+    <section class="second">
+        <div class="container2">
+            <p>"This is an inspiring quote or a testimonial from a customer. Maybe it's just filling up space, or maybe people will actually read it. Who knows? All I know is that it looks nice.</p>
+            <h1>-Thor, God of Thunder</h1>
```

Comment:
`<h1>` is typically meant to represent the top level header of the page.

For this page that is the "This Website is Awesome".

I think for this you could just do a `<b>` or a `<span>` that you style